doc/4-Merge.md

```diff
@@ -63,15 +63,74 @@ class Analyser_Merge_Charging_station_FR(Analyser_Merge):
 
 ## OpenData set Source
 
-The OpenData set is available online and can be fetched by a URL. On edge case, the data can be included directly on the Osmose-QA backed repository, but we try to avoid this case.
+The OpenData set is available online and can be fetched online. On edge case, the data can be included directly on the Osmose-QA backed repository, but we try to avoid this case.
 
-The OpenData set have this attributes:
-- URL of the Data, included in an archive or not:
+There are multiple OpenData classes to facilitate retrieving data, depending on the source.
+
+### SourceOpenDataSoft
+A lot of OpenData providers use [OpenDataSoft](https://www.opendatasoft.com/). It can usually be identified by the presence of `/explore/dataset/` in the URL.
+
+It uses the following attributes:
+- `url`: The URL of the dataset
+- `format`: Requested format of the file. Usually `csv` (default), `json` or `shp`
+
+Example:
+
+```python
+from .Analyser_Merge import SourceOpenDataSoft
+
+    SourceOpenDataSoft(
+        attribution='La Poste',
+        url='https://datanova.legroupe.laposte.fr/explore/dataset/laposte_boiterue',
+    )
+```
+
+### SourceDataGouv
+Data shared on [data.gouv.fr](https://data.gouv.fr/) can be retrieved with this class.
+
+It uses the following attributes:
+- `dataset`: Dataset identifier. It can be seen by clicking on Details on the dataset page.
+- `resource`: The resource (file) identifier. It can be seen by clicking on the resource in the dataset page. It's the end of the "stable URL"
```

Comment:
```suggestion
- `resource`: The resource (file) identifier. It can be seen by clicking on the resource in the dataset page. It's the end of the "stable URL".
```